File: app/src/main/java/me/jy/danggi/activities/model/Memo.java
package me.jy.danggi.activities.model;

import java.io.Serializable;
import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Created by JY on 2018-01-12.
 */
@Data
@AllArgsConstructor(staticName = "of")
@NoArgsConstructor
public class Memo implements Serializable{

    private String content;
    private Date writeDate;


}

File: app/src/main/java/me/jy/danggi/data/Memo.kt
package me.jy.danggi.data

import io.realm.RealmObject
import io.realm.RealmResults
import io.realm.annotations.LinkingObjects
import io.realm.annotations.PrimaryKey
import java.io.Serializable
import java.util.*

open class Memo(
        @PrimaryKey
        var id: String = UUID.randomUUID().toString(),

        var content: String = "",

        var writeDate: Date = Date()

) : RealmObject(), Serializable